### DISCUSSION TITLE
Add an option for CA Certificates

### DISCUSSION THREAD

**Discussion by tygoee** [cockpit-project/cockpit#20870] [Q&A]
Page: Network

Could it be possible to add an option for CA Certificates to login, so users need to install them to access cockpit, and then essentially automatically login. Other people trying to access the page won't be able to access it without the private key.

This would both be more secure and (after installation) more practical; not having to login

Most devices natively support them, so accessing the page on a Mac, Linux, Android or iOS device shouldn't be an issue

**Comment by martinpitt** [comment]
Yes, this is supported. See https://cockpit-project.org/guide/latest/cert-authentication for the documentation. I also did a demo how to set up smart card auth (which is essentially the same) here: https://www.youtube.com/watch?v=v07cHZQsRyo
